pact.cabal

```diff
@@ -27,6 +27,11 @@ flag ghc-flags
   default:     False
   manual:      True
 
+flag build-tool
+  description: Include modules and deps for tests + executable
+  default:     True
+  manual:      False
```

Comment:
Should this be `True`? It seems that `default` is only used when `manual: False`. I think, cabal lets the solver choose the value when `manual: False`. Currently, it seems that cabal always disable this flag.